README.md

```diff
@@ -20,10 +20,10 @@ Git tags are used to mark minor and patch release versions.
 
 ## Reading the latest version of the documentation
 
-The latest version of the P4Runtime v1 specification is available:
-* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.html) in **HTML**
+The latest version of the P4Runtime v1.3 specification is available:
+* [here](https://p4lang.github.io/p4runtime/spec/v1.3.0/P4Runtime-Spec.html) in **HTML**
   format
-* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.pdf) in **PDF**
+* [here](https://p4lang.github.io/p4runtime/spec/v1.3.0/P4Runtime-Spec.pdf) in **PDF**
   format
 
 It is updated every time a new commit is pushed to the main branch.
```

Comment:
This is supposed to be the version generated from the main branch:

> It is updated every time a new commit is pushed to the main branch.

Reply:
Hi, this was intended as a temporary fix, at least until you solved the issues with the p4.org links. Should I just close this pull request?

Reply:
I think the PR is fine, but the links should be:
* https://p4lang.github.io/p4runtime/spec/main/P4Runtime-Spec.html
* https://p4lang.github.io/p4runtime/spec/main/P4Runtime-Spec.pdf